tests/core/numerics/interpolator/test_main.cpp

```diff
@@ -632,7 +634,7 @@ class ACollectionOfParticles_1d : public ::testing::Test
             part.v[2]     = +1.;
             particles.push_back(part);
         }
-        interpolator(makeIndexRange(particles), rho, v, layout);
+        interpolator(makeIndexRange(particles), rho, rho_c, v, layout);
```

Comment:
is there some check that could be added to the tests `DepositCorrectlyTheirWeight_*d` 

[eg](https://github.com/PHAREHUB/PHARE/blob/8473002ed2d85ffb7a1fa8b7efc27711fb51436b/tests/core/numerics/interpolator/test_main.cpp#L654-L657)